src/strategies/dfyn-staked-in-farms/index.ts

```diff
@@ -0,0 +1,145 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'vatsalgupta13';
+export const version = '0.1.0';
+
+
+const DFYN_SUBGRAPH_URL = {
+  '137': 'https://api.thegraph.com/subgraphs/name/ss-sonic/dfyn-v5'
+};
+
+const getAllLP = async (skip, stakedLP, network) => {
+  let params =
+  {
+    pairs: {
+      __args: {
+        where: {
+          id_in: stakedLP.map((address) => address.toLowerCase())
```

Comment:
In this query you should add a way to query specific block number if the snapshot is not "latest". Currently it may return incorrect result.

Reply:
Thanks for pointing this out. I have made the required changes.